server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -871,6 +871,7 @@ exports.updateClient = async function (body, user, xCorrelator, traceIndicator,
     currentReleaseNumber,
     newReleaseFwName);
 
+if((currentApplicationName == futureApplicationName || currentApplicationName !== futureApplicationName) && currentReleaseNumber !== futureReleaseNumber){
```

Comment:
this condition **(currentApplicationName == futureApplicationName || currentApplicationName !== futureApplicationName)** will return true for all cases, hence it shall be reformed.